fenix/app/src/main/res/drawable-v23/splash_screen.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:drawable="@mipmap/ic_launcher_round"
+        android:width="160dp"
+        android:height="160dp"
```

Comment:
dimensions from the splash screen [doc](https://developer.android.com/develop/ui/views/launch/splash-screen#dimensions)